.runUnit/Unit Tests.run.xml

```diff
@@ -0,0 +1,25 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Is it an IntelliJ tests run definition? Haven't seen such folder before.

Reply:
yep :)